test/extended/util/annotate/rules.go

```diff
@@ -231,6 +231,21 @@ var (
 		"[Skipped:NoOptionalCapabilities]": {
 			// This test requires a valid console url which doesn't exist when the optional console capability is disabled.
 			`\[sig-cli\] oc basics can show correct whoami result with console`,
+			// These tests require the ImageRegistry which is disabled. Most these tests will either directly interact with
+			// ImageRegistry or will test things such as ImageStreams, some test functionality in other Operators such as
+			// Controller Manager Operator (uses ImageRegistry to create .dockercfg for SAs), which rely on ImageRegistry
+			`\[sig-imageregistry\]\[Feature:ImageAppend\] Image append should create images by appending them`,
+			`\[sig-imageregistry\]\[Feature:ImageExtract\] Image extract should extract content from an image`,
+			`\[sig-imageregistry\]\[Feature:ImageInfo\] Image info should display information about images`,
+			`\[sig-imageregistry\]\[Feature:Image\] oc tag should work when only imagestreams api is available`,
+			`\[sig-imageregistry\] Image registry \[apigroup:route.openshift.io\] should redirect on blob pull`,
+			`\[sig-imageregistry\]\[Feature:ImageLookup\] Image policy should update standard Kube object image`,
+			`\[sig-imageregistry\]\[Feature:ImageLookup\] Image policy should perform lookup when the object has the resolve-names`,
+			`\[sig-imageregistry\]\[Feature:ImageTriggers\] Annotation trigger reconciles after the image is overwritten`,
+			`\[sig-devex\]\[Feature:OpenShiftControllerManager\] TestAutomaticCreationOfPullSecrets \[apigroup:config.openshift.io\]`,
+			`\[sig-devex\]\[Feature:OpenShiftControllerManager\] TestDockercfgTokenDeletedController \[apigroup:image.openshift.io\]`,
```

Comment:
Unfortunately, these tests just make sure the `.dockercfg` is created in the SA pull secret, working around that would make the tests moot so I thought we might as well skip them. 

It looked like OCM was the one managing these when I looked, I come across this [loop in OCM](https://github.com/openshift/openshift-controller-manager/blob/master/pkg/serviceaccounts/controllers/create_dockercfg_secrets.go#L498-L505) when I was tacking this down. Those `dockerURLs` get populated through service listers that look for registries in [this array](https://github.com/openshift/openshift-controller-manager/blob/master/pkg/serviceaccounts/controllers/docker_registry_service.go#L51-L54)

Reply:
That's why I asked what's happening, ie. what kind of pull secrets are being created when the registry is disabled. Would be good to followup and check that, because as I see it, we either create secrets with dummy data, or we don't create anything at all. The latter would be the expected outcome, but based on my quick look, I haven't seen that happening in OCM. If the former (dummy data), that's even worse, b/c we're providing something we should not. In either case, this deserves a followup.

Reply:
Absolutely, I went ahead and created a Jira to track that effort https://issues.redhat.com/browse/OCPVE-731